code-matlab/shared/datatypes/tc/tc.m

```diff
@@ -1,84 +1,50 @@
 function tc_out = tc(varargin)
 %% TC TUNING CURVE DATATYPE
+%   The tc dataype is a container for tuning curves typically used in experimental
+%   analysis.
+%
 %   tc_out = TC(varargin) is a constructor for the tc (tuning curve) struct.
 %
-% youkitan 2014-10-27
+%   INPUTS:
+%
```

Comment:
1) OK
2) right, so if this new `tc` isn't backwards compatible (is it?) should create an issue